### ISSUE TITLE
Configuration file is created in parent folder

### ISSUE THREAD

**Issue by juliangrube1988** [opticdev/optic#2256]
When i create a new optic.yml file via `optic capture init openapi/api.json` the optic.yml was created in the parent folder and not the current folder.

**To Reproduce**
1. install optic 0.49.3
2. place api spec in openapi/api.json
3. create optic.yml: `optic capture init openapi/api.json`
4. now optic.yml was created in parent folder

**Expected behavior**
`optic capture init openapi/api.json` should create the file in the folder it was run in or there should be the option to pass the path to the optic.yml as a argument

**Details :**
 - OS: Debian 11.7
 - Optic: 0.49.3
 - NodeJS version: 18.14.0

**Additional context**

**Comment by acunniffe** [comment]
Thanks for opening this issue -- we'll check it out. Is the parent folder the root of your git repository?

**Comment by juliangrube1988** [comment]
Yes the parent folder is the root of the git repository.

**Comment by notnmeyer** [comment]
hi @juliangrube1988, what you're describing is our intended behavior—there is a single optic config file per repo and we place it in the repo's root. for `capture` at least, there is support for multiple openapi specs. for example,

```yaml
capture:
  myapi/openapi.yml:
    server: ...
  myapi2/openapi.yml:
    server: ...
```

so running `optic capture myapi/openapi.yml` or `optic capture myapi2/openapi.yml` should work if you had a monorepo.

can you share a bit more about the repo you're using optic in, or why you'd like to specify where the optic configuration lives so we can better understand your use case?

**Comment by juliangrube1988** [comment]
We are using optic in a repository containing the front- and backend. Both are in their own folder and have a workspace definition.

**Comment by niclim** [comment]
Hi @juliangrube1988 - we released `0.49.9` which will now detect the closest `optic.yml` from the current working directory. 

By default, Optic is going to assume you want the `optic.yml` in the root of the project folder (`optic.yml` supports having multiple capture configurations for different folders), but you can override this behavior by creating an empty `optic.yml` file in the folder you want to initialize the repo in.

In your case, you can:
- Upgrade your optic version
- Create an empty `optic.yml` file in the `/frontend` and `/backend` folders
- Run `optic capture init` in the `/frontend` and `/backend` folders
